src/polyfill/bigint64array.js

```diff
@@ -0,0 +1,89 @@
+// BigInt64Array polyfill for Safari versions between v14.0 and v15.0.
+// All browsers other than Safari added BigInt and BigInt64Array at the same
+// time, but Safari introduced BigInt in v14.0 and introduced BigInt64Array in
+// v15.0
+
```

Comment:
Wrap this whole file in `if (typeof BigInt64Array === "undefined") {`

Reply:
Okay, done.